### PR TITLE
 ✨ banner integration with API, client changes

### PR DESCRIPTION
**Pull request by alexchomiak**
✨Adding banner api integration into microservice API, Client

### CODE REVIEW

File: api/src/routes/banner/index.ts [hunk 1]
```diff
@@ -17,16 +17,19 @@ router.post('/subscribe', isAuthenticated, async (req: Request, res: Response) =
...
         await User.updateOne({ _id }, { $addToSet: { subscriptions: crn } })
-
+        console.log('SUBSCRIBING WITH ' + `http://localhost:4000/notify/${_id}/${crn}`)
         // * Subscribe via Banner API
         try {
             // * Waiting for Banner Client to be completely implemented
-            await BannerClient.post('/subscribe', {
-                hook: `/notify/${_id}/${crn}`
+            await BannerClient.post('/hook', {
+                url: `http://localhost:4000/notify/${_id}/${crn}`,
```
Comment (bmiddha) [inline api/src/routes/banner/index.ts:25]: remove hardcoded origin

File: api/src/routes/banner/index.ts [hunk 2]
```diff
@@ -47,13 +50,16 @@ router.post('/unsubscribe', isAuthenticated, async (req: Request, res: Response)
 
         // * Subscribe via Banner API
         try {
+            console.log('UNSUBSCRIBING WITH ' + `http://localhost:4000/notify/${_id}/${crn}`)
```
Comment (bmiddha) [inline api/src/routes/banner/index.ts:53]: remove hardcoded origin

File: client/package.json [hunk 3]
```diff
@@ -70,7 +70,9 @@
         "redux-persist": "^6.0.0",
         "redux-thunk": "^2.3.0",
         "regenerator-runtime": "^0.13.5",
-        "tiny-warning": "^1.0.3"
+        "tiny-warning": "^1.0.3",
+        "react-select": "~3.1.0",
+        "@types/react-select": "~3.0.11"
```
Comment (bmiddha) [inline client/package.json:75]: move to dev-deps

**Review by bmiddha — APPROVED** [review]
🌮 Left some comments inline.
Other than that. It looks good!

**Review by anarmawala — APPROVED** [review]
🌮 Looks good, left a comment!

**Comment by alexchomiak** [comment]
Thanks guys! Everything is starting to come together. 🌮

**Pull request merged**